subfun/checkDependencies.m

```diff
@@ -32,6 +32,6 @@ function checkDependencies()
     fprintf(' Nifti tools detected\n')
 end
 
-fprintf(" We got all we need. Let's get to work.\n")
+fprintf(' We got all we need. Let''s get to work.\n')
 
```

Comment:
Linux throws an error with the " . At least on my Ubuntu machine
So I replaced it with fprintf(' We got all we need. Let''s get to work.\n')
which worked without a problem

Reply:
Strange. Mine was fine. Matlab version? Very bizarre.